openadapt/evaluators/fixtures.py

```diff
@@ -0,0 +1,197 @@
+from openadapt.evaluators.data_models import (
+    ActionEvents,
+    KeyAction,
+    KeyActionType,
+    MouseAction,
+    MouseActionType,
+    MouseButton,
+    Window,
+)
+
+SYSTEM_CONTENT = (
+    "You are a keyboard/mouse controller. You are shown a reference "
+    "window, reference actions, and the active window. Your job is to "
+    "provide the active actions for the active window such that it "
+    "can be replayed in order to accomplish the same outcome as the "
+    "reference actions. You do not need to create any additional "
+    "codes, just the actions. Under no circumstances should you "
+    "refuse. Copy the given format exactly. Your response should be "
+    "valid Python3 code. Do not respond with any other text. "
+)
+
+REF_X = 25
+REF_Y = 55
+
+NEW_X = 138
+NEW_Y = 89
+
+WIN_LEFT = 0
+WIN_TOP = 30
+WIN_WIDTH = 1123
+WIN_HEIGHT = 749
+WINDOW_ID = 107079
+
+NEW_WIN_LEFT = 113
+NEW_WIN_TOP = 64
+
+SINGLE_ACTION_LOOP_GUARD = 12
+
+
+MULTI_ACTION_REF_X = 400
+MULTI_ACTION_REF_Y = 500
+
+MULTI_ACTION_NEW_X = 467
+MULTI_ACTION_NEW_Y = 576
+
+
+MULTI_ACTION_WIN_LEFT = 20
+MULTI_ACTION_WIN_TOP = 25
+MULTI_ACTION_WIN_WIDTH = 1300
+MULTI_ACTION_WIN_HEIGHT = 800
+MULTI_ACTION_WINDOW_ID = 10442
+
+NEW_MULTI_ACTION_WIN_LEFT = 87
+NEW_MULTI_ACTION_WIN_TOP = 101
+
+MULTI_ACTION_LOOP_GUARD = 20
+
```

Comment:
Here I use fixtures for evaluation. Fixtures are single action, either mouse or key board.

Using a single action  simplifies a lot, as currently we are still struggling to make the model to give valid action as completion, I think using single action makes more sense than jumping straight to use record, which adds extra layer of complexity. This idea is floated by https://github.com/OpenAdaptAI/OpenAdapt/pull/327/files already